homeassistant/components/hue.py

```diff
@@ -105,9 +102,10 @@ def setup(hass, config):
 
     for bridge in bridges:
         filename = bridge.get(CONF_FILENAME)
-        allow_unreachable = bridge.get(CONF_ALLOW_UNREACHABLE)
-        allow_in_emulated_hue = bridge.get(CONF_ALLOW_IN_EMULATED_HUE)
-        allow_hue_groups = bridge.get(CONF_ALLOW_HUE_GROUPS)
+        allow_unreachable = bridge.get(CONF_ALLOW_UNREACHABLE,
```

Comment:
Pass defaults because when discovering bridges via nupnp, we don't go through config schema.